var/spack/repos/builtin/packages/catalyst/package.py

```diff
@@ -51,23 +55,62 @@ class Catalyst(CMakePackage):
     variant('python', default=False, description='Enable Python support')
     variant('essentials', default=False, description='Enable Essentials support')
     variant('extras', default=False, description='Enable Extras support')
-    variant('rendering', default=False, description='Enable Vtk Rendering support')
+    variant('rendering', default=False, description='Enable VTK Rendering support')
+    variant('expat', default=True, description="Use external expat")
```

Comment:
If this is disabled, does it use an internal copy? If so, we should always enable this, no need for a variant.

Reply:
There is in internal vtkexpat, but for this particular paraview/VTK version we'd need to dig out a patch. It was also broken for me (out of the box).

Regarding other patches.
- The vtkm one has already been patched upstream, but got forgotten (again) for paraview 5.5.2.
- The catalyst patches (handling of working directory) are something that Andy Bauer has already merged in upstream, but won't be rolled out until 5.6. Nonetheless a bit of a showstopper for our use of catalyst.
- The change in environment paths for paraview 5.5 are something that we (spack) needs to accommodate. In older versions libraries were dropped into lib/paravew-5.4 etc, now they are simply under lib/.